### ISSUE TITLE
make it possible to get the genesis file from outside the classpath

### ISSUE THREAD

**Issue by adridadou** [ethereum/ethereumj#594]
It looks like Genesis can be loaded only by loading from the classpath.

The code I have found is:
```
public static Genesis loadGenesis(SystemProperties config, ClassLoader classLoader)  {
        String genesisFile = config.genesisInfo();

        InputStream is = classLoader.getResourceAsStream("genesis/" + genesisFile);
        return loadGenesis(config, is);
    }
```

The issue here is that you cannot load the genesis json file from:
1. Outside the classpath. For example if someone wants to provide it during runtime
2. As a generic InputStream. It would be nice to be able to give it as a String, a remote connection Stream etc ....

**Comment by Nashatyrev** [comment]
I believe this issue is partially fixed with #514 (i.e. specify via file path)

**Comment by adridadou** [comment]
cool! 
I'll close this issue and I'll see if I need to do a PR for my use case or not
